build-aux/snap/snapcraft.yaml

```diff
@@ -24,15 +24,17 @@ parts:
   foliate:
     plugin: meson
     source: .
+    override-pull: |
+      snapcraftctl pull
+      sed -i -e 's|@GJS@|/usr/bin/gjs|g' src/com.github.johnfactotum.Foliate.in
```

Comment:
The sed hack is currently needed because meson sets the path to gjs using `$PATH` at build time. The gnome-3-34 extension puts the build snap’s path first in $PATH. So meson replaces @GJS@ with /snap/gnome-3-34-1804-sdk/current/usr/bin/gjs for the interpreter path. This is the correct path in the build VM, but will not work in the the actual snap.

We'd like to find a way to fix this without this `sed` patch.